conf/v1.routes

```diff
@@ -1,3 +1,5 @@
 PUT        /sample/:nino/:taxYear       v1.controllers.AmendSampleController.amendSample(nino: String, taxYear: String)
 GET        /sample/:nino/:taxYear       v1.controllers.RetrieveSampleController.retrieveSample(nino: String, taxYear: String)
 DELETE     /sample/:nino/:taxYear       v1.controllers.DeleteSampleController.deleteSample(nino: String, taxYear: String)
+
```

Comment:
Uri is missing `annual`in the url 

Example Uri is `individuals/business/self-employment/{nino}/{businessId}/annual/{taxYear}`